test_conformance/subgroups/subgroup_common_templates.h

```diff
@@ -394,6 +396,22 @@ void genrand(Ty *x, Ty *t, cl_int *m, int ns, int nw, int ng)
 {
```

Comment:
I'd recommend changing the function name to something more specific to test_subgroups, like gen_inputs or such. genrand makes it look like a random number generator, but it's much more specialized.

Reply:
done